src/blockchain_txn_token_convert_v1.proto

```diff
@@ -0,0 +1,15 @@
+syntax = "proto3";
+package helium;
+
+import "blockchain_token_type_v1.proto";
+
+message blockchain_txn_token_convert_v1 {
+    blockchain_token_type_v1 token_type = 1;
```

Comment:
is this always a one-way conversion, like HNT -> X? else how does it indicate what token is being converted into what other token?